apps/essence/server/services/db/repository/product.ts

```
@@ -0,0 +1,35 @@
+      .update(products)
+      .set(data)
+      .where(eq(products.id, id))
+      .returning()
+
+    return product
+  }
+
+  static async delete(id: string) {
+    return useDatabase().delete(products).where(eq(products.id, id))
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)